web/src/App.less

```diff
@@ -124,6 +124,7 @@ img {
   flex: auto;
   display: flex;
   align-items: center;
+  justify-content: center;
```

Comment:
Why add this?

Reply:
PromptPage will in left if don't add this. But it break other pages position config. Now need further adjustment.